### ISSUE TITLE
wrong forward daily returns ?

### ISSUE THREAD

**Issue by luca-s** [quantopian/alphalens#74]
I noticed the daily forward returns are calculated as the forward returns divides by the forward period  (see [utils.compute_forward_returns](https://github.com/quantopian/alphalens/blob/master/alphalens/utils.py#L57)) .

 I don't exactly understand the reason so I might be wrong, but shouldn't the daily returns calculated as:

```
# let's pretend those are the variables
# 'days' is our period
# 'fwd_ret' is the return after the period

# the current implementation does something like the following (what does this value mean?)
daily_returns = fwd_ret / days

# while I am suggesting to calculate the daily returns as the value the returns would have
# had every single day for the whole period if they had grown at a steady rate
daily_returns = (fwd_ret+1)**(1./days) - 1

```

**Comment by twiecki** [comment]
Yes, your method would be correct. Want to do a PR that fixes it?

**Comment by luca-s** [comment]
Sure, I will.

**Comment by luca-s** [comment]
PR #75

**Comment by twiecki** [comment]
Fixed by #75 .
